src/System.Numerics.Vectors/src/System/Numerics/Matrix3x2.cs

```diff
@@ -802,9 +802,9 @@ public override string ToString()
         /// <returns>The hash code.</returns>
         public override int GetHashCode()
         {
-            return M11.GetHashCode() + M12.GetHashCode() +
-                   M21.GetHashCode() + M22.GetHashCode() +
-                   M31.GetHashCode() + M32.GetHashCode();
+            return unchecked(M11.GetHashCode() + M12.GetHashCode() +
```

Comment:
Fair point, but honestly I don't know what to standardize on. The style of the files I've changed so far varies widely so I've tried to pick something that I thought fit with the rest of the file in each case. Suggestions?